<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1415</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>108</width>
      <height>236</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="squar">
       <property name="text">
        <string>Квадрат</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rect">
       <property name="text">
        <string>Прямоугольник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="paral">
       <property name="text">
        <string>Параллелограмм</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="romb">
       <property name="text">
        <string>Ромб</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="treugl">
       <property name="text">
        <string>Треугольник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="trapc">
       <property name="text">
        <string>Трапеция</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="circle">
       <property name="text">
        <string>Круг</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sector">
       <property name="text">
        <string>Сектор</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="img_circle">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/circle.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img_paral">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>350</y>
      <width>231</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/paral.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img_rect">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>391</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/rect.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img_romb">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>360</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/romb.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img_sec">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/sec.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img_square">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/square.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img_trap">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>370</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/trap.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img_treg">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/treg.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>1010</width>
      <height>252</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_25">
       <item>
        <layout class="QVBoxLayout" name="circle_form">
         <item>
          <layout class="QVBoxLayout" name="layout_square_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_17">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_19">
                 <item>
                  <widget class="QLabel" name="label_r_circle">
                   <property name="text">
                    <string>Радиус</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_r_circle"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_21">
             <item>
              <widget class="QPushButton" name="result_circle">
               <property name="text">
                <string>Посчитать</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="area_circle">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="formula_circle">
           <property name="text">
            <string>Площадь круга = Радиус**2 * pi</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="paral_form">
         <item>
          <layout class="QVBoxLayout" name="layout_square_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_height_paral">
                   <property name="text">
                    <string>Высота</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_height_paral"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_width_paral">
                   <property name="text">
                    <string>Ширина</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_width_paral"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QPushButton" name="result_paral">
               <property name="text">
                <string>Посчитать</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="area_paral">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="formula_paral">
           <property name="text">
            <string>Площадь параллелограмма = Высота * Ширина</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="rect_form">
         <item>
          <layout class="QVBoxLayout" name="layout_square_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_length_rect">
                   <property name="text">
                    <string>Длина</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_length_rect"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_width_rect">
                   <property name="text">
                    <string>Ширина</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_width_rect"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="result_rect">
               <property name="text">
                <string>Посчитать</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="area_rect">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="formula_rect">
           <property name="text">
            <string>Площадь прямоугольника = Длина * Ширина</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="romb_form">
         <item>
          <layout class="QVBoxLayout" name="layout_square_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                  <widget class="QLabel" name="label_d1_romb">
                   <property name="text">
                    <string>1 диагональ</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_d1_romb"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_11">
                 <item>
                  <widget class="QLabel" name="label_d2_romb">
                   <property name="text">
                    <string>2 диагональ</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_d2_romb"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QPushButton" name="result_romb">
               <property name="text">
                <string>Посчитать</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="area_romb">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="formula_romb">
           <property name="text">
            <string>Площадь ромба =(1 диагональ * 2 диагональ)/2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_24">
       <item>
        <layout class="QVBoxLayout" name="sec_form">
         <item>
          <layout class="QVBoxLayout" name="layout_square_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_18">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_19">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_20">
                 <item>
                  <widget class="QLabel" name="label_d_sec">
                   <property name="text">
                    <string>Дуга</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_d_sec"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_22">
                 <item>
                  <widget class="QLabel" name="label_r_sec">
                   <property name="text">
                    <string>Радиус</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_r_sec"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_23">
             <item>
              <widget class="QPushButton" name="result_sec">
               <property name="text">
                <string>Посчитать</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="area_sec">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="formula_sec">
           <property name="text">
            <string>Площадь сектора = (Дуга * Радиус)/2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="trap_form">
         <item>
          <layout class="QVBoxLayout" name="layout_square_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_15">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_16">
                 <item>
                  <widget class="QLabel" name="label_height_trap">
                   <property name="text">
                    <string>Высота</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_height_trap"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_17">
                 <item>
                  <widget class="QLabel" name="label_width_trap">
                   <property name="text">
                    <string>Средняя линия</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_width_trap"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QPushButton" name="result_trap">
               <property name="text">
                <string>Посчитать</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="area_trap">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="formula_trap">
           <property name="text">
            <string>Площадь трапеции = Высота * Средняя линия</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="suare_form">
         <item>
          <layout class="QVBoxLayout" name="layout_square">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QLabel" name="label_length_square">
                   <property name="text">
                    <string>Длина</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_length_square"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_width_square">
                   <property name="text">
                    <string>Ширина</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_width_square"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="result_square">
               <property name="text">
                <string>Посчитать</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="area_square">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="formula_square">
           <property name="text">
            <string>Площадь квадрата = Длина * Ширина</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="treg_form">
         <item>
          <layout class="QVBoxLayout" name="layout_square_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13">
                 <item>
                  <widget class="QLabel" name="label_height_treg">
                   <property name="text">
                    <string>Высота</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_height_treg"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_14">
                 <item>
                  <widget class="QLabel" name="label_width_treg">
                   <property name="text">
                    <string>Прилег. сторона</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="inp_width_treg"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QPushButton" name="result_treg">
               <property name="text">
                <string>Посчитать</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="area_treg">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="formula_treg">
           <property name="text">
            <string>Площадь треугольника = (Высота * Прилег. сторона) / 2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1415</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="authors">
    <property name="title">
     <string>Авторы</string>
    </property>
   </widget>
   <widget class="QMenu" name="clear">
    <property name="title">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QMenu" name="exit">
    <property name="title">
     <string>Выход</string>
    </property>
   </widget>
   <addaction name="authors"/>
   <addaction name="clear"/>
   <addaction name="exit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
